space_packet_parser/xtce/comparisons.py

```diff
@@ -389,21 +383,8 @@ def evaluate(self,
 
         def _get_parsed_value(parameter_name: str, use_calibrated: bool):
             """Retrieves the previously parsed value from the passed in packet"""
-            try:
-                return packet[parameter_name] if use_calibrated \
-                    else packet[parameter_name].raw_value
-            except KeyError as e:
-                raise ComparisonError(f"Attempting to perform a Condition evaluation on {self.left_param} but "
-                                      "the referenced parameter does not appear in the hitherto parsed data passed to "
-                                      "the evaluate method. If you intended a comparison against the raw value of the "
-                                      "parameter currently being parsed, unfortunately that is not currently supported."
-                                      ) from e
-
-        # TODO: Consider allowing one of the parameters to be the parameter currently being evaluated.
```

Comment:
I thought this did work since I'm using it in other places already? Maybe that is only in a different "Comparison"... Or is the "parameter_name" the same as `self`'s name in this case so it works because of that? Ultimately I'm wondering if we can get rid of the `current_parsed_value` since it is ignored in several places and somehow rely on the comparison to the value in the packet.

Reply:
I'm going to add a TODO item in place of this removed error handling with a reference to a new ticket.

Reply:
See #163